src/interfaces/IBCoWPool.sol

```diff
@@ -67,4 +70,102 @@ interface IBCoWPool is IERC1271 {
    * AMM has not enabled trading.
    */
   error AppDataDoNotMatchHash();
+
+  /**
+   * @notice Once this function is called, it will be possible to trade with
+   * this AMM on CoW Protocol.
+   * @param appData Trading is enabled with the appData specified here.
+   */
+  function enableTrading(bytes32 appData) external;
+
+  /**
+   * @notice Disable any form of trading on CoW Protocol by this AMM.
+   */
+  function disableTrading() external;
+
+  /**
+   * @notice Restricts a specific AMM to being able to trade only the order
+   * with the specified hash.
+   * @dev The commitment is used to enforce that exactly one AMM order is
+   * valid when a CoW Protocol batch is settled.
+   * @param orderHash the order hash that will be enforced by the order
+   * verification function.
+   */
+  function commit(bytes32 orderHash) external;
+
+  /**
+   * @notice The value representing the absence of a commitment.
+   * @return _emptyCommitment The commitment value representing no commitment.
+   */
+  function EMPTY_COMMITMENT() external view returns (bytes32 _emptyCommitment);
```

Comment:
I added `solhint-disable-next-line`. I think is cleaner, wdyt?

Reply:
I think it makes sense to disable that part of the rule for the entire directory since there are lots of external functions that'll return constants, and for interfaces already defined by someone else we can't do anything about it and have to live with the names they chose. 

The config file being in `src/interfaces` means we won't disable the rule anywhere else

Reply:
Would there be a case where we don't want to disable the rule under `src/interfaces`, and it would catch a bad practice?

I don't think it's cleaner to disable the rule on a line-per-line basis on every line where it'd report

Not dying on this hill tho, doesn't block merging

Reply:
> Would there be a case where we don't want to disable the rule under src/interfaces, and it would catch a bad practice?

hmm yeah I think it's possible. For example, [this](https://github.com/defi-wonderland/world-id-governor-core/blob/dev/src/interfaces/IGovernorWorldID.sol#L106) is another project where we do it like that

Reply:
shall we move the constants to a `BCoWConst` contract?
similar to their implementation, we make the test import the constants and we can make use of them (w/o requiring the interface)

Reply:
hmm okey. I can move them back. But im curious whats the point in having the constants outside? it's cheaper in gas?